README.md

```diff
@@ -11,12 +11,26 @@ Use it by setting the extends field in your `.eslintrc`:
 };
 ```
 
-## Prettier
+## For adverity-components 
 
-With prettier support:
+```js
+{
+  extends: '@pmedianetwork/eslint-config/components',
+};
+```
+
+## For insights
+
+```js
+{
+  extends: '@pmedianetwork/eslint-config/insights',
+};
+```
+
+## For datatab 
 
 ```js
 {
-  extends: '@pmedianetwork/eslint-config/prettier',
+  extends: '@pmedianetwork/eslint-config/datatab',
```

Comment:
datatap

Reply:
ups

Reply:
https://github.com/pmedianetwork/eslint-config/pull/3/commits/2be5625f996ff0373ad2a08ad29b4a72a2323eed

Reply:
the comment above it also has this typo

Reply:
https://github.com/pmedianetwork/eslint-config/pull/3/commits/a6e871e7ffacf54e397a2b97d59e30b9fdc0012c